templates/zerver/help/invite-new-users.md

```diff
@@ -11,9 +10,8 @@ the article below describes each in more detail.
 
 * Share a **reusable invitation link**.
 
-The last two, invite-based, techniques also allow you to control the
-[role (owner, admin, moderator, member, or guest)](/help/roles-and-permissions) that the
-invited people will have.
+The last two, invite-based techniques also allow you to control the
+[role](/help/roles-and-permissions) that the invited people will have.
```

Comment:
Let's make this a little easier to read. Something like:

> When sending an invitation, you can also control the
[role](/help/roles-and-permissions) that the invited users will have when they join.